### PR TITLE
Updated docstrings for pymc.Beta

### PR DESCRIPTION
**Pull request by sagartomar**
**Thank your for opening a PR!**

Depending on what your PR does, here are a few things you might want to address in the description:
+ [ ] what are the (breaking) changes that this PR makes?
+ [ ] important background, or details about the implementation
+ [ ] are the changes—especially new features—covered by tests and docstrings?
+ [x] [linting/style checks have been run](https://docs.pymc.io/en/latest/contributing/python_style.html)
+ [ ] [consider adding/updating relevant example notebooks](https://github.com/pymc-devs/pymc-examples)
+ [ ] right before it's ready to merge, mention the PR in the RELEASE-NOTES.md

For issue #5459

#DataUmbrellaPyMCSprint

### CODE REVIEW

File: pymc/distributions/continuous.py [hunk 1]
```diff
@@ -1204,14 +1205,14 @@ class Beta(UnitContinuous):
...
     ----------
-    alpha: float
-        alpha > 0.
-    beta: float
-        beta > 0.
-    mu: float
-        Alternative mean (0 < mu < 1).
-    sigma: float
-        Alternative standard deviation (0 < sigma < sqrt(mu * (1 - mu))).
+    alpha : tensor_like of float
```
Comment (OriolAbril) [inline pymc/distributions/continuous.py]: alpha and beta are also optional I think, but I don't know what the defaults are @ricardoV94

Comment (sagartomar) [reply pymc/distributions/continuous.py]: If we don't specify alpha, beta then they will be calculated using mu, sigma. Does it make sense to write the equation for computing alpha, beta from mu, sigma in the docstrings or will that be too long?

Comment (OriolAbril) [reply pymc/distributions/continuous.py]: the formula is already there in the extended summary (at the bottom)

Comment (sagartomar) [reply pymc/distributions/continuous.py]: is it okay then to just change it to: `alpha : tensor_like of float, optional` or should we add something in the description saying that if not specifed then calculated from mu and sigma?

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**